optaplanner-core/src/main/java/org/optaplanner/core/api/score/holder/ScoreHolder.java

```diff
@@ -103,7 +113,9 @@
      * @return never null
      * @throws IllegalStateException if {@link #isConstraintMatchEnabled()} returns false
      * @see ScoreDirector#getIndictmentMap()
+     * @deprecated for removal from public API
```

Comment:
Let's reference ScoreManager because if they're using this, they probably want to learn about ScoreManager.